stix2/observables.py

```diff
@@ -213,11 +368,54 @@ def _check_object_constaints(self):
         self._check_at_least_one_property(["src_ref", "dst_ref"])
 
 
+class WindowsProcessExt(_STIXBase):
+    _properties = {
+        'aslr_enabled': BooleanProperty(),
+        'dep_enabled': BooleanProperty(),
+        'priority': StringProperty(),
+        'owner_sid': StringProperty(),
+        'window_title': StringProperty(),
+        'startup_info': DictionaryProperty(),
+    }
```

Comment:
Should have an object constraint - all properties are optional but at least one of them must be present.